querybook/tests/test_lib/test_export/test_exporters/test_gspread_exporter.py

```diff
@@ -57,7 +55,7 @@ def test_get_no_columns(self):
         # edge case - 0 columns (ie no statement results) shouldn't break this function
         self.statement_columns_len_mock.return_value = 0
         exporter = GoogleSheetsExporter({})
-        self.assertEqual(exporter._get_max_rows(1), MAX_SHEETS_CELLS)
+        self.assertEqual(exporter._get_max_rows(1), MAX_SHEETS_CELLS // 26)
```

Comment:
i think in this case we can just assert its not zero and there is no exception, there is no need to check the divide by 26 logic (unless you want to have a test case specifically for that)